test/TestConfig/scripts/build_test.xml

```diff
@@ -61,7 +61,7 @@
 	<macrodef name="checkGitRepoSha">
 		<attribute name="repoDir" default="." />
 		<sequential>
-			<exec executable="bash" failonerror="false">
+			<exec executable="bash" failonerror="true">
```

Comment:
Not sure if we should make this change. It depends on if the expected SHA information isn't available should we stop the build process or just ignore it.